daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/CidContainer.scala

```diff
@@ -74,10 +74,7 @@ trait CidContainer[+A] {
     suffixer.traverse[String] {
       case Value.ContractId.V1(discriminator, Bytes.Empty) =>
         Value.ContractId.V1.build(discriminator, f(discriminator))
-      case acoid @ Value.ContractId.V1(_, _) =>
-        Right(acoid)
-      case acoid @ Value.ContractId.V0(_) =>
```

Comment:
Fyi @simonmaxen-da @danielporterda @vivd-da - it looks like in the corda code base you also use `suffixCid` but assume that you are working with V1 contract ids only. From what I could tell (see description of this PR) this should not affect daml-on-corda.

Reply:
That is correct.  We don't have an existing install base so all our ContractIds will only ever be V1 is my understanding.

Reply:
Thank you for confirming, Sir!